package.json

```diff
@@ -0,0 +1,63 @@
+{
+  "name": "jaeger-client-javascript",
+  "version": "0.1.0",
```

Comment:
Note: there needs to be a way for this lib to know its own version, so that it can report it in `Span.Process.Tags['jaeger.version']`